interactions:
- request:
    body: '{"scope":"test:testcanceldowntimesbyscopereturnsdowntimesnotfoundresponse1633076912_invalid"}'
    headers:
      accept:
      - application/json, */*;q=0.8
      connection:
      - close
      content-length:
      - '93'
      content-type:
      - application/json
      host:
      - api.datadoghq.com
      user-agent:
      - datadog-api-client-typescript/1.0.0-beta.5 (node 14.18.0; os Linux; arch x64)
      x-datadog-parent-id:
      - '3703768329805907252'
      x-datadog-trace-id:
      - '1612844427823424916'
    method: POST
    uri: https://api.datadoghq.com/api/v1/downtime/cancel/by_scope
  response:
    body:
      string: '{"errors": ["No downtimes found"]}'
    headers:
      cache-control:
      - no-cache
      connection:
      - close
      content-length:
      - '34'
      content-security-policy:
      - frame-ancestors 'self'; report-uri https://api.datadoghq.com/csp-report
      content-type:
      - application/json
      date:
      - Fri, 01 Oct 2021 08:28:32 GMT
      pragma:
      - no-cache
      strict-transport-security:
      - max-age=15724800;
      vary:
      - Accept-Encoding
      x-content-type-options:
      - nosniff
      x-frame-options:
      - SAMEORIGIN
    status:
      code: 404
      message: Not Found
version: 1
